pkg/morph/client/multi.go

```diff
@@ -120,6 +132,79 @@ func (c *Client) notificationLoop() {
 	}
 }
 
+func (c *Client) switchToMostPrioritized() {
+	const period = 30 * time.Second
```

Comment:
I think 30 second is a really small time for this kind of thing, something like 10 minutes is more suitable IMO. cc @realloc

Reply:
I'd suggest moving all similar constants to a config file anyway.

Reply:
Agree, but the default still should make sense.

Reply:
From the test cases, we often see 2 minutes as a requirement.

Reply:
changed to 2 min

> moving all similar constants to a config file anyway

should it be done in that PR? not sure if it is useful. mb add when it has use cases?

Reply:
It is definitely useful, because we must be able to control such things.

Reply:
added